telegram/_chat.py

```diff
@@ -585,10 +905,30 @@ def __init__(
         self.business_location: Optional["BusinessLocation"] = business_location
         self.business_opening_hours: Optional["BusinessOpeningHours"] = business_opening_hours
 
+        if self.__class__.__name__ == "Chat":
+            for arg in _deprecated_attrs:
+                if (val := object.__getattribute__(self, arg)) is not None and val != ():
```

Comment:
what is the `val != ()` for? is that for array-type attributes that can be empty sequences?

Reply:
yep